src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -5470,13 +5470,33 @@ class Goo<T>
 }
 ";
 
-            var compilation = CreateCompilation(source);
+            var compilation = CreateCompilation(source, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7));
             compilation.VerifyDiagnostics(
                 // (2,2): error CS0404: Cannot apply attribute class 'Goo<T>' because it is generic
                 // [Goo<int>]
                 Diagnostic(ErrorCode.ERR_AttributeCantBeGeneric, "Goo<int>").WithArguments("Goo<T>").WithLocation(2, 2));
```

Comment:
>ERR_AttributeCantBeGeneric [](start = 37, length = 26)

This diagnostic is wrong. It should be a LangVer diagnostic, or `ERR_NotAnAttributeClass`, or maybe both?